### ISSUE TITLE
Tool tips broken on org model and non org model tables

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#3152]
### Current Behavior

Currently if you try to click a tool tip on any table, your mouse icon will show as a question mark and no tool tip will show.

### Expected Behavior

Tool tips should show the tooltip text when a user hovers over the info icon
This applies to org model and non org model

### Steps to Reproduce

1. Go the Domains table
2. try to hove over the "i" icon
3. note that the tool tip doesn't show

### Environment

stable and all other envs

### Additional Context

note [this open uswds bug ticket ](https://github.com/uswds/uswds/issues/3871)

**Comment by erinysong** [comment]
Since this issue's status hasn't changed on USWDS's end, do we want to go with the same approach as the poster of the linked issue and remove the disabled class on the button containing the tooltip then modify the button ourselves to manually disable it?

**Comment by erinysong** [comment]
Slacked USWDS's public support channel to ask on the status of that tagged issue

**Comment by erinysong** [comment]
Added our use case to [USWDS #5190](https://github.com/uswds/uswds/issues/5190) since someone else has also reported tooltips aren't supported across non-button elements which goes against their docs that all elements can have tooltips.
